client/state/ui/post-type-list/reducer.js

```diff
@@ -20,37 +12,14 @@ import {
 const initialState = {
 	postIdWithActiveSharePanel: null,
 	postIdWithActiveLikesPopover: null,
-	isMultiSelectEnabled: false,
-	selectedPosts: [],
 };
 
 export const postTypeList = ( state = initialState, action ) => {
 	switch ( action.type ) {
-		case POST_TYPE_LIST_MULTI_SELECTION_MODE_TOGGLE:
-			return {
-				...state,
-				isMultiSelectEnabled: ! state.isMultiSelectEnabled,
-				selectedPosts: state.isMultiSelectEnabled ? state.selectedPosts : [],
-			};
-
-		case POST_TYPE_LIST_SELECTION_TOGGLE:
-			if ( state.selectedPosts.indexOf( action.postGlobalId ) > -1 ) {
-				return {
-					...state,
-					selectedPosts: without( state.selectedPosts, action.postGlobalId ),
-				};
-			}
-			return {
-				...state,
-				selectedPosts: [ ...state.selectedPosts, action.postGlobalId ],
-			};
-
 		case ROUTE_SET:
 			return {
 				...state,
-				isMultiSelectEnabled: false,
 				postIdWithActiveLikesPopover: null,
```

Comment:
This Likes Popover state should be local to the Post List component. Not a good fit for Redux. It's not related to this PR though, I'm just complaining here about the "put everything into Redux" era 🙂

Reply:
Couldn't agree more! I'll make a note to follow-up and clean this area up further.

Reply:
Followed up with this in #45293.